rdr_service/offline/main.py

```diff
@@ -888,6 +889,22 @@ def nph_sms_n1_generation():
     return '{"success": "true"}'
 
 
+@app_util.auth_required_cron
+@check_genomic_cron_job('tactis_participants_to_bq')
```

Comment:
While I like that you are using this decorator we wrote, I can see the use-case now for extending it outside the realm of genomics, but I think it might be better to change the schedule in cron_default.yaml to `schedule: 1 of jan 12:00` and remove the use of this decorator since its specific to Genomics workflows, that way it nevers get executed and we have scripts for updating the cron scheduler when tactis is ready, and you can add a subsequent PR to the cron schedule to make it permanent.

Reply:
Updated!